package.json

```diff
@@ -164,7 +164,10 @@
   "resolutions": {
     "**/**/ws": ">=7.4.6",
     "**/**/json-bigint": ">=1.0.0",
-    "**/**/mongoose": "~5.10.14"
+    "**/**/mongoose": "~5.10.14",
+    "mpath": "^0.8.4",
+    "lodash": "^4.17.12",
+    "lodash.merge": "^4.6.2"
```

Comment:
@bodymindarts Do you think this is the correct way to resolve criticals deps?
I have added the versions as [the audit](https://ci.galoy.io/teams/dev/pipelines/galoy-app/jobs/audit/builds/2#L62309a8a:5) reported.

Reply:
This seems like a hack, this is being done by Justin properly in other PRs.